spec/lib/post_action_creator_spec.rb

```diff
@@ -6,6 +6,10 @@
   fab!(:post) { Fabricate(:post) }
   let(:like_type_id) { PostActionType.types[:like] }
 
+  before do
+    Group.refresh_automatic_groups!
```

Comment:
I chose to selectively apply this to the test that need it, I found automatically doing it for all tests e.g. in the user fabricator was a disaster because there are many tests relying on specific group counts, and we also don't want this huge overhead for tests that don't need it.